### PR TITLE
added decomposed meshes

### PR DESCRIPTION
**Pull request by ichumuh**
- added convex decomposition obj for all collision objects
- changed urdfs to load objs as collsion objects
- bring up launch file has a static transform publisher for map
- bring up launch file has a joint state publisher
- wardrobe_door_left_joint opens in positive directions

### CODE REVIEW

File: iai_apartment/launch/apartment_display.launch
Comment (gaya-) [inline iai_apartment/launch/apartment_display.launch:6]: Can you call it iai_kitchen? For consistency. Maybe we should rename to something proper but I can't be bothered now :)

File: iai_apartment/launch/apartment_bringup.launch
Comment (gaya-) [inline iai_apartment/launch/apartment_bringup.launch:24]: This one too /iai_kitchen/cram_joint_states

File: iai_apartment/urdf/kitchen.xarco
Comment (gaya-) [inline iai_apartment/urdf/kitchen.xarco:34]: Hmm, I don't mind removing Bond James Bond comments (although Lara might get upset :( ) but the cabinet comments might be useful. I guess this was to be expected with a script... I wonder if we can make these comments into a proper tag so it wouldn't disappear.

File: iai_apartment/launch/apartment_display.launch
Comment (ichumuh) [reply iai_apartment/launch/apartment_display.launch:6]: I don't care what the topic is called. Thought the convention was to name it after the package.

Comment (gaya-) [reply iai_apartment/launch/apartment_display.launch:6]: No, it's a hard-coded string. So please change if you don't care.

**Comment by ichumuh** [comment]
renamed the topics

**Comment by mir7b** [comment]
@ichumuh it would be great if you could fit my new merge request stuff in here as well! XD

File: iai_apartment/urdf/kitchen.xarco
Comment (mir7b) [inline iai_apartment/urdf/kitchen.xarco:805]: this comment, for example, will be very useful in future. I'd argue for the sake of maintainability, some of these comments must stay.

File: iai_apartment/urdf/kitchen.xarco
Comment (mir7b) [inline iai_apartment/urdf/kitchen.xarco:1391]: also very useful comment. using keywords so maintainers can do quick search.

File: iai_apartment/urdf/kitchen.xarco
Comment (mir7b) [inline iai_apartment/urdf/kitchen.xarco:34]: This comment serves a purpose. This file is currently (new version) about 1750 lines long. Either this should turn into a hell of nested xacro files, or these comments serve as folding points. Without them, you can't easily fold and compare the snippets of code. I suggest you add something to your script to ignore the comments altogether. if you want to change individual comments because you think they're not good or don't explain enough, by all means, go ahead. but removing the structural ones, just makes any further maintenance a pain.

**Comment by ichumuh** [comment]
I'm using a different xml parser which should preserve all comments, does this look good now? @gaya- @mir7b

**Comment by gaya-** [comment]
Looks good to me. I'd merge it after #99 is merged and this PR is updated with #99 contents.

**Review by mir7b — APPROVED** [review]
seems good!

**Comment by ichumuh** [comment]
@gaya- @mir7b merged with master

**Pull request merged**
